core/src/main/scala/com/raphtory/graph/visitor/Vertex.scala

```diff
@@ -379,6 +379,15 @@ trait Vertex extends EntityVisitor {
   def isInNeighbour(id: Long): Boolean  = inNeighbourSet.contains(id)
   def isOutNeighbour(id: Long): Boolean = outNeighbourSet.contains(id)
 
+//  def removeInNeighbour(id: Long)
```

Comment:
These a TODO for later?

Reply:
These are Stubbs from thinking about options for doing edge filtering (though probably not how we actually should implement it)